packages/components/badge/stories/Badge.stories.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 import { Flex } from '@contentful/f36-core';
 
-import { Badge } from '../src/Badge';
+import { Badge, _Badge } from '../src/Badge';
```

Comment:
Why is this necessary if it's the same component?

Reply:
Apparently, `__docgenInfo` is not parsed properly if a component is wrapped in `forwardRef` 🤕 

As an alternative, we can do `export const BadgeDocgenInfo = _Badge['__docgenInfo']` from Badge.tsx

Reply:
I see. I like the alternative as it's clearer why it's done 🙂

Reply:
An alternative approach didn't work :(

Reply:
Oh I'm too late for this

I know you've merged this already, but I think we don't need that
```
parameters: {
    propTypes: [Badge['__docgenInfo']],
  },
```
It was used for generating the propTypes table back when we were not using storybook docs

Now that we use docs, it generates a prop table in the Docs table automagically without that parameter. Look:
![Screenshot 2021-03-17 at 10 49 10](https://user-images.githubusercontent.com/6597467/111448411-ac9a0900-870e-11eb-9dac-deb498cc7e73.png)

It even gets JSDoc comments as descriptions of the props ☝️ 

Long story short, I think we can remove the docgenInfo thing for every new component and then we don't need the _Badge thing 🙌